docs/enterprise/helm-guide.md

```diff
@@ -0,0 +1,228 @@
+feldera-db-0                                 1/1     Running   0          5m31s
+feldera-kubernetes-runner-6447b8f56d-86j4w   1/1     Running   0          5m31s
+feldera-prober-server-5c945b7fcf-9r727       1/1     Running   0          5m31s
+feldera-compiler-server-0                    1/1     Running   0          5m31s
+feldera-api-server-c546499bc-wdpkm           1/1     Running   0          5m31s
+```
+
+**StatefulSets:**
+```
+$ kubectl get sts -n feldera
+NAME                      READY   AGE
+feldera-db                1/1     5m41s
+feldera-compiler-server   1/1     5m41s
+```
+
+**Deployments:**
+```
+$ kubectl get deployments -n feldera
+NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+feldera-kubernetes-runner   1/1     1            1           5m55s
+feldera-prober-server       1/1     1            1           5m55s
+feldera-api-server          1/1     1            1           5m55s
+```
+
+**Persistent volume claims:**
+```
+$ kubectl get pvc -n feldera
+NAME                                         STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+db-storage-feldera-db-0                      Bound    pvc-a5acb4ef-1d3c-4773-ad42-528657f11c94   20Gi       RWO            local-path     6m11s
+compiler-storage-feldera-compiler-server-0   Bound    pvc-07369993-c20a-4f8a-abd1-52bf3be413c3   20Gi       RWO            local-path     6m11s
+```
+
+### Uninstallation
+
+The most direct and thorough way to uninstall Feldera can be done
+if it was installed in its own dedicate namespace by deleting the
+namespace itself: `kubectl delete namespace feldera`
+
+If you wish to preserve the namespace, it can be uninstalled through
+the usual `helm` method:
+```
+helm uninstall -n feldera feldera
+```
+
+This does not delete the persistent volumes, which need to be deleted separately:
+```
+kubectl get pvc -n feldera
+kubectl delete pvc -n feldera db-storage-feldera-db-0
+kubectl delete pvc -n feldera compiler-storage-feldera-compiler-server-0
+kubectl get pvc -n feldera
+```
+
+## Troubleshooting
+
+### Incorrect license
+
+If an incorrect account id and/or license key was provided, the kubernetes-runner
+will fail to start:
+
+```
+$ kubectl get pods -n feldera
+NAME                                         READY   STATUS             RESTARTS     AGE
+feldera-db-0                                 1/1     Running            0            2m34s
+feldera-compiler-server-0                    1/1     Running            0            2m34s
+feldera-kubernetes-runner-854446fb84-rg66z   0/1     CrashLoopBackOff   1 (9s ago)   2m34s
+feldera-api-server-7bb757f685-jwn5v          1/1     Running            0            2m34s
+feldera-prober-server-698c999557-f8nsr       1/1     Running            0            2m34s
+```
+
+... and an explanation will be visible in its log:
+
+```
+$ kubectl logs -n feldera feldera-kubernetes-runner-854446fb84-rg66z
```

Comment:
For logs, it's easier to query the deployment directly. That way, the instructions here can be copy-pasted:

`kubectl logs -n deployment/feldera-kubernetes-runner`